File: Frontend/angular/src/app/model/address.ts

export class Address
{
    houseNo : string;
    street : string;
    city : string;
    pinCode : string;

    constructor(houseNo:any, street:any, city:any, pinCode:any)
    {
        this.houseNo = houseNo;
        this.street = street;
        this.city = city;
        this.pinCode = pinCode;
    }
}

File: FinalCapstoneProject/Backend/FoodieApp/userRestaurantService/src/main/java/com/example/userRestaurantService/Model/Address.java
package com.example.userRestaurantService.Model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document
@NoArgsConstructor
@AllArgsConstructor
@Data
public class Address
{
    @Id
    private String houseNo;
    private String street;
    private String city;
    private String pinCode;
}
